nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -478,11 +479,12 @@ NavfnPlanner::getCostmap(
   // TODO(orduno): explicitly provide specifications for costmap using the costmap on the request,
   //               including master (aggregate) layer
 
-  auto request = std::make_shared<nav2_tasks::CostmapServiceClient::CostmapServiceRequest>();
-  request->specs.resolution = 1.0;
+  nav2_world_model::CostmapServiceRequest request;
+  request.specs.resolution = 0.05;
```

Comment:
Why the change in resolution?

Reply:
Using a more reasonable value. -- although it's currently ignored when servicing the request.